### PR TITLE
Use vf-animations mixin across all stylesheets

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Use vf-animations mixin across all stylesheets

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Ensure no errors

### CODE REVIEW

File: src/components/Layout/_layout.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "vanilla-framework/scss/settings_colors";
+@import "vanilla-framework/scss/vanilla";
 @import "../../scss/settings";
```
Comment (anthonydillon) [inline src/components/Layout/_layout.scss]: Don't you want settings imported before vanilla so it can override variables.

Comment (barrymcgee) [reply src/components/Layout/_layout.scss]: They don't currently override any Vanilla settings but I suppose they may in future - I've now swapped them around.

File: src/components/Terminal/_terminal.scss [hunk 2]
```diff
@@ -36,7 +40,6 @@ $terminal-header-height: 40px;
 
     .is-visible & {
       transform: rotate(0);
```
Comment (anthonydillon) [inline src/components/Terminal/_terminal.scss]: Doesn't this need a animation?

Comment (barrymcgee) [reply src/components/Terminal/_terminal.scss]: There's another issue to remove this anyway.

**Pull request merged**